tests/tests/createmany-parallel-mounts.sh

```diff
@@ -21,6 +21,9 @@ START=$SECONDS
 createmany -o "$T_D0/file_" $COUNT >> $T_TMP.full
 sync
 SINGLE=$((SECONDS - START))
+# It's possible on fast devices that a single createmany takes
+# under 1 second. That messes up the test math further on.
+if (( SINGLE==0 )); then (( SINGLE=1 )) ; fi
```

Comment:
As you say, please figure out something better.  Increasing the count might be the least intrusive.  Next best would be adding a little test helper that returns milliseconds since the script started.  That gets higher precision without floats or large values that are a source of errors.  Looping increasing the load in the test until it gets a usably large measurement is probably too much.

Reply:
No need to write a whole helper, we can just rely on `bc` and `date` to do this. See updated commit.